tasks/configure.yml

```diff
@@ -142,7 +143,7 @@
     src: etc_systemd_system_postgresql.service.d_custom.conf.j2
     dest: "/etc/systemd/system/postgresql-{{ postgresql_version }}.service.d/custom.conf"
   when: ansible_os_family == "RedHat"
-  register: postgresql_systemd_custom_conf
```

Comment:
This `postgresql_systemd_custom_conf` variable doesn't seem to be used anywhere else, so I renamed it to be more generic... so that I could also attach it to the Debian code at line 50.